client/src/components/Search/Search.js

```diff
@@ -445,74 +418,76 @@ class Search extends Component {
 
     // Convert the tag data to a plain array and remove un-needed attributes.
     return tagDataCopy ?
-      Object.values(tagDataCopy).map(({ key, label, value }) => ({ key, label, value })) :
+      Object.values(tagDataCopy).map(({ key: tagKey, label, value }) => ({ key: tagKey, label, value })) :
```

Comment:
What is the `key: tagKey` bit here for? It doesn't look like it's necessary?

Reply:
Fixed